src/WalkingUtils.js

```diff
@@ -0,0 +1,64 @@
+//@flow
+
+function parseWalkingRoute(data: any, startDateMs: number) {
+    let path = data.paths[0];
+    let endDateMs = startDateMs + path.time;
+
+    let departureTime = new Date(startDateMs).toISOString().split('.')[0]+"Z";
+    let arrivalTime = new Date(endDateMs).toISOString().split('.')[0]+"Z";
+
+    let boundingBox = {
+        minLat: path.bbox[1],
+        minLong: path.bbox[0],
+        maxLat: path.bbox[3],
+        maxLong: path.bbox[2]
+    };
+
+    let startCoords = {
+        lat: path.points.coordinates[0][1],
+        long: path.points.coordinates[0][0]
+    };
+
+    let endCoords = {
+        lat: path.points.coordinates[path.points.coordinates.length - 1][1],
+        long: path.points.coordinates[path.points.coordinates.length - 1][0]
+    };
+
+    let numberOfTransfers = 0
+
+    let walkingPath = path.points.coordinates.map(point => {
```

Comment:
does this path contain more than two coordinates for proper map matching? or do we need to add the average lat/long again?